src/phoenix/server/api/routers/v1/spans.py

```diff
@@ -50,11 +50,17 @@ async def query_spans_handler(request: Request) -> Response:
               start_time:
                 type: string
                 format: date-time
+                nullable: true
               stop_time:
                 type: string
                 format: date-time
+                nullable: true
+              limit:
+                type: integer
+                nullable: true
```

Comment:
can the default be placed in the openAPI spec? 

e.x. 
```
limit
integer · min: 1 · max: 100 · default: 100
Integer numbers.
```

Reply:
Added the default. This PR does not implement a `min` and `max` value. Going to punt on that since I assume it will be covered when we introduce pagination.